styles/main.scss

```diff
@@ -17,6 +17,10 @@
 }
 
 [data-theme='dark'] {
+  // Override react-tostify variable here for easy theme adherence
+  --toastify-color-light: #121212;
```

Comment:
This colour is the default toastify one, I tried the usual bg secondary (since main background is usually the primary) and am not sure if it stands out enough as a notification element on top of the page that way

Reply:
Good point, let's keep it like that then.

Reply:
Ah, you already changed it, looks good to me.